tests/framework_tests/test_analytics.py

```diff
@@ -24,20 +27,20 @@ def test_get_total_activity_count(self):
         date = datetime.utcnow()
 
         assert_equal(analytics.get_total_activity_count(user._id), 0)
-        assert_equal(analytics.get_total_activity_count(user._id), user.get_activity_points(db=self.db))
+        assert_equal(analytics.get_total_activity_count(user._id), user.get_activity_points(db=None))
```

Comment:
I should probably just remove the db kwarg from all the functions, I was just trying to keep the API as similar as possible to lower the chance of mistakes.